components/strava/actions/get-stats/get-stats.mjs

```diff
@@ -0,0 +1,17 @@
+import strava from "../../strava.app.js";
+
+export default {
+  name: "Get Stats",
+  description: "Returns the activity stats of an athlete. Only includes data from activities set to Everyone visibilty. See `https://developers.strava.com/docs/reference/`",
```

Comment:
Could you please use []() to ref the docs?

Reply:
done